webpack/assets/javascripts/react_app/components/UserJwtToken/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { get } from '../../redux/API';
+import { USER_JWT_TOKEN } from './UserJwtTokenConstants';
+import UserJwtTokenForm from './components/UserJwtTokenForm';
+import TokenField from './components/TokenField';
+import { selectStatus, selectUserJwtToken } from './UserJwtTokenSelectors';
+
+const UserJwtToken = ({ userId }) => {
+  const dispatch = useDispatch();
+  const apiStatus = useSelector(selectStatus);
```

Comment:
you can use the API selector directly IMO 

```suggestion
  const apiStatus = useSelector(state => selectAPIStatus(state, USER_JWT_TOKEN));
```